collect_app/src/main/java/org/odk/collect/android/formentry/FormEntryUseCases.kt

```diff
@@ -29,21 +29,28 @@ object FormEntryUseCases {
         formsRepository: FormsRepository,
         projectRootDir: File,
         formDefCache: FormDefCache
-    ): Pair<FormDef, Form> {
+    ): Pair<FormDef?, Form?> {
```

Comment:
Could this return `Pair<FormDef, Form>?` instead and then the check on the other side could just be if the whole result is null?

Reply:
I was thinking about that but thought it wouldn't be better... but maybe https://github.com/getodk/collect/pull/5835/commits/fed5b7de5134201df0bcbf6651885d993fa79b31
If you like it more we can keep it.